java-generator/core/src/main/java/io/fabric8/java/generator/nodes/JavaNameAndType.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.java.generator.nodes;
+
+public class JavaNameAndType {
```

Comment:
Lombok-ify?

```
@Getter
@RequiredArgsConstructor
```

Reply:
I feel this should also be comparable and have a toString, so
```
@ToString
@EqualsAndHashCode
```

Reply:
I don't think that introducing `lombok` annotations, at this stage of the project, will significantly improve this baseline.

Reply:
Yes, please do not introduce more dependencies unless strictly required.